certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
```

Comment:
My apologies for missing the call to discuss this, but I don't agree with the `authorized` word here at all for end entity certificates.

What it means to be "authorized" will vary from agency to agency, and isn't something we can control. We do not want to be subject to audit risk if an agency fails to manage its internal policy controls correctly.

We only want to be subject to audit for failing to meet domain validation controls. If an "unauthorized" organization representative with control of a DNS name obtains a certificate from this PKI for that DNS name, then the agency might be sad internally, but it was not a failure of this PKI.

Reply:
it's not DoD or civilian - it's a we statement.  "we"..."we"...

I'm going to **draft** remove the full portion of:  "an authorized organization representative".  I don't know what an organization is _in this context_ and by definition except "organization=U.S. Government".

Reply:
I get that it is "we" - but in the manual process - it is not the subscriber (device) or the RA - "authorized organization representative" is what "we" call a "PKI Sponsor" on Common/FBCA CP, DoD CP.  And that IS who submits the requests.  The CA (directly or via an RA) needs to determine who is "authorized" - DoD does this today and needs to continue to be able to do it for the foreseeable future in the MANUAL process.  My concern - WHEN we write the CPS to say that is what we are going to do - will it be judged non-compliant.  If the answer is YES - NON-COMPLIANT, then the phrase needs to REMAIN.  As a policy geek who does "compliance analysis" for DoD, I would argue that it is NOT complaint as the CP says one of two entities and you are now saying a third.   And, yes, the "organization is "US Government" - all the authorized organizatio0n representatives" will be Civ, Mil or contractors working for part of the US Government.  I for one will recommend a critical comment on this issue when it comes around for review - so we can decide how to fix it now or do it in adjudication.

Section 3.2.2 says the CA verifies "and the authenticity of the Applicant Representative's certificate request" - that is the "authorized organization representative".  In DoD, the PROCESS is for the RA to work with the organization directly to determine that the PERSON who submitted the request was authorized to do so.  I suspect every other organization which does manual NPE cert issuance has a very similar process.

Reply:
@LarryFrank 

It's a bit simpler for an answer.  I - for one - am against mixing terms that aren't defined and want to stay away from "Sponsor".   

But you pointed out the answer = align with 3.2.2.  So instead of "authorized organization representative" which has no direct correlation to any other terms yet used - why not use "Applicant representative"?

Reply:
That works for me - as long as we have some way for a person to submit the request...

Reply:
done! Applicant representative...(now I need to go change my architecture diagrams to say "applicant representative" :smile: )

https://github.com/uspki/policies/pull/137/commits/3e62ca76f16d7f5fbed0082f671b0c155f2652fd